src/main/java/de/tum/in/www1/artemis/service/ResultService.java

```diff
@@ -64,17 +62,6 @@ public ResultService(UserRepository userRepository, ResultRepository resultRepos
         this.ratingRepository = ratingRepository;
     }
 
-    /**
-     * Sets the assessor field of the given result with the current user and stores these changes to the database. The User object set as assessor gets Groups and Authorities
-     * eagerly loaded.
-     *
-     * @param result Result for which current user is set as an assessor
-     */
-    public void setAssessor(Result result) {
-        User currentUser = userRepository.getUser();
-        result.setAssessor(currentUser);
-    }
-
```

Comment:
Why did you remove this code? Was it unused? Same question for the block below in the same file.

Reply:
Yes it's not used